rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { shallowMount } from "@vue/test-utils";
+import RundeckInfo from "./RundeckInfo.vue";
+
+jest.mock("../../../rundeckService", () => ({
+  getRundeckContext: jest.fn().mockReturnValue({ rdBase: "mocked-rdBase" }),
+  url: jest.fn().mockReturnValue("mocked-url"),
+}));
+
+const mountRundeckInfo = async (props?: Record<string, any>) => {
+  return shallowMount(RundeckInfo, {
+    props: {
+      appInfo: { title: "Rundeck", logocss: "some-css" },
+      ...props,
+    },
+  });
+};
+describe("RundeckInfo", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("renders RundeckLogo when appInfo.title is 'Rundeck'", async () => {
+    const wrapper = await mountRundeckInfo({});
+    expect(wrapper.findComponent({ name: "RundeckLogo" }).exists()).toBe(true);
+    expect(wrapper.findComponent({ name: "PagerdutyLogo" }).exists()).toBe(
+      false,
+    );
+  });
+  it("renders PagerdutyLogo when appInfo.title is 'Pagerduty'", async () => {
+    const wrapper = await mountRundeckInfo({ appInfo: { title: "Pagerduty" } });
+
+    expect(wrapper.findComponent({ name: "RundeckLogo" }).exists()).toBe(false);
+    expect(wrapper.findComponent({ name: "PagerdutyLogo" }).exists()).toBe(
+      true,
+    );
+  });
+  it("renders RundeckVersions information correctly ", async () => {
+    const versionProps = {
+      logo: false,
+      number: "1.0.0",
+      tag: "stable",
+    };
+    const wrapper = await mountRundeckInfo({ version: versionProps });
+    const versionComponent = wrapper.findComponent({ name: "RundeckVersion" });
+    expect(versionComponent.exists()).toBe(true);
```

Comment:
2 things:

- the text you write in the "it" is meant to describe what the test does. In this case "renders information correctly". To check this, it's not enough to only assert if that component exists, especially since this component will actually generate the text it will display by combining the properties from the versionProps.

- secondly, you should check for the final dom elements, in this case, find the span inside versionComponent and check that the text generated is what it should be.